### PR TITLE
Removed dead link to old wiki

### LINKED ISSUE #815: Link to the old wiki is now dead
At the bottom of this page:

https://github.com/umbraco/UmbracoDocs/blob/master/Reference/Templating/Macros/Xslt/45-XML-Schema.md

is a link referring to the old wiki, which now is dead:

`/wiki/reference/xslt/45-xml-schema/switching-between-old-and-new-schema`

I haven't been able to find a replacement in the new documentation - hence this issue.

image

### PR DESCRIPTION
**Pull request by andreathniah**
Closes #815

### CODE REVIEW

**Comment by dampee** [comment]
Hi @andreathniah 
Do you have a profile on http://our.umbraco.com?

File: Reference/Templating/Macros/Xslt/45-XML-Schema.md [hunk 1]
```diff
@@ -66,8 +66,8 @@ XSLT file from the old to the new schema:
 
 However if you wish to revert back to the old XML Schema you need to
 update the ```UseLegacyXmlSchema``` setting in the
-```config/UmbracoSettings.config``` file, [make sure to read this wiki
+```config/UmbracoSettings.config``` file, make sure to read this wiki
```
Comment (dampee) [inline Reference/Templating/Macros/Xslt/45-XML-Schema.md]: I think we need more reprashing as the old wiki can not be found.  So you can not read the wiki page.  Removing the link alone is not enough.  :-)

Comment (andreathniah) [reply Reference/Templating/Macros/Xslt/45-XML-Schema.md]: @dampee Do you think its better to just remove the entire section on reverting back to the old schema? Given that this "new" schema was introduced in Umbraco 4.5, it has been nearly 8 years (we're now in v7.12!) and I highly doubt there are people who have yet to update. Not to mention I can't find any reverting instructions/docs to replace this link 🤔

**Comment by andreathniah** [comment]
> Hi @andreathniah
> Do you have a profile on http://our.umbraco.com?

Yes, I do have an [account](https://our.umbraco.com/member/275493). Hope the new amendment is okay!

**Comment by dampee** [comment]
Thanks a lot @andreathniah 

I'll request your contributor badge for your OUR profile.

**Comment by nul800sebastiaan** [comment]
Hi there @andreathniah,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing :pray:

If you haven't heard already we have had a record breaking amount of PR's in both the CMS and the Documentation repository. You can read all about the numbers in the [Umbraco Hacktoberfest Aftermath blog post](https://umbraco.com/blog/the-2018-hacktoberfest-aftermath/).

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://goo.gl/forms/smjQwEMDM5cUnz6x1) :point_left: Please follow this link to fill out and submit the form, **before November 21st 2018**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before December, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Pull request merged**